build.gradle

```diff
@@ -1,19 +1,36 @@
-apply plugin: 'com.android.library'
-
-group = 'org.slamon.android'
```

Comment:
Actually, I'm not sure. It used to be 'org.slamon.android' but I don't know how it fairs if I change it to just 'org.slamon'.